web-registry/src/components/organisms/BasketsTable.tsx

```diff
@@ -33,26 +42,32 @@ export const BasketsTable: React.FC<{
         >
           Asset
         </Box>,
-        'Balance',
+        'Amount available',
       ]}
       rows={baskets.map((row, i) => {
         return [
           <Grid container wrap="nowrap">
             <Grid item>
               <BasketIcon />
             </Grid>
-            <Grid item sx={{ ml: 3.5, fontWeight: 700 }} alignSelf="center">
+            <Grid
+              item
+              sx={{ ml: 3.5, pb: 2, fontWeight: 700 }}
+              alignSelf="center"
+            >
               <Box sx={{ fontSize: theme => theme.spacing(4.5) }}>
-                {row.name}
+                {row.basket.name}
```

Comment:
thanks for the comments Mark! I'm resolving these conversations about undefined, though I'm not making any changes to this PR, because I'm incorporating them into the next PR, which is based on this one and adds some refactoring, etc.